docs/protocol/evmos-cli/configuration.mdx

```diff
@@ -1,38 +1,17 @@
-# Evmos CLI
+---
+sidebar_position: 1
+---
 
-`evmosd` is the all-in-one command-line interface. It supports wallet management, queries and transaction operations 
+# Configuration
 
-## Prerequisite Readings
+TODO Only include configurations:
 
-- [Installation](./installation)
+* Node Configuration
+  * pruning
+* Client Configuration
+  * command options
```

Comment:
I am not able to comment on non-changed code but if you search for the code part, you can locate this:

1. if you search for `10token`, should token be aevmos or Evmos?
2. if you search `---pruning everything` is there one extra dash to the flag?
3. I think in app.toml line, we should add links to things like telemetry, SDK, etc.

Reply:
As described in the description of this PR, these files don't need to be reviewed.